### ISSUE TITLE
[Feature] Info in console if there's not enough ETH

### ISSUE THREAD

**Issue by AdamB2023** [tsarbuig/WalletHunter#36]
It appears that there's no info in console about not enough WETH.
Swaps below haven't had ID and wasn't processed.

```
-- SWAP DETECTED -----------------------------------------------
02-13 12:43:51.4351  Tx TX_ID - id: 0a4d                    
                                                                
-- SWAP DETECTED -----------------------------------------------
02-13 12:44:14.4414  Tx TX_ID - id: 813e        
02-13 12:44:14.4414  813e - This Tx is an approval Tx --> ignore
                                                                
-- SWAP DETECTED -----------------------------------------------
02-13 12:48:35.4835  Tx TX_ID - id: be42                                                                
```

I had it for another swap:
`BALANCE CHECK -------- Your wallet hold 0 of token Wrapped Ether --> not enough to make BUY order --> let's stop here`

**Comment by tsarbuig** [comment]
send me logs in PM please, I'm not sure to understand

**Comment by tsarbuig** [comment]
solved in [v0.49.5](https://github.com/tsarbuig/WalletHunter/releases/tag/v0.49.5)
